tests/v3/test_indexing.py

```diff
@@ -1782,9 +1782,9 @@ async def test_accessed_chunks(
 
         # Combine and generate the cartesian product to determine the chunks keys that
         # will be accessed
-        chunks_accessed = []
-        for comb in itertools.product(*chunks_per_dim):
-            chunks_accessed.append(".".join([str(ci) for ci in comb]))
+        chunks_accessed = [
+            ".".join([str(ci) for ci in comb]) for comb in itertools.product(*chunks_per_dim)
+        ]
```

Comment:
Maybe we can simplify this a bit further?

```suggestion
        chunks_accessed = [
            ".".join(map(str, comb)) for comb in itertools.product(*chunks_per_dim)
        ]
```

Please see PR: https://github.com/zarr-developers/zarr-python/pull/2229